README.rst

```diff
@@ -47,34 +63,19 @@ To build a complete set of HTML documentation, you must have Sphinx, which can b
 
 The built html files can be found in doc/_build/html afterward.
 
-
-Twisted
-=======
-
-Eventlet had Twisted hub in the past, but community interest to this integration has dropped over time,
-now it is not supported, so with apologies for any inconvenience we discontinue Twisted integration.
-
-If you have a project that uses Eventlet with Twisted, your options are:
-
-* use last working release eventlet==0.14
-* start a new project with only Twisted hub code, identify and fix problems. As of eventlet 0.13, `EVENTLET_HUB` environment variable can point to external modules.
-* fork Eventlet, revert Twisted removal, identify and fix problems. This work may be merged back into main project.
-
-Apologies for any inconvenience.
-
 Supported Python versions
 =========================
 
 Python 3.7-3.12 are currently supported.
 
-Flair
-=====
-
-.. image:: https://img.shields.io/pypi/v/eventlet
-    :target: https://pypi.org/project/eventlet/
+Project Guideline
+=================
 
-.. image:: https://img.shields.io/github/actions/workflow/status/eventlet/eventlet/test.yaml?branch=master
-    :target: https://github.com/eventlet/eventlet/actions?query=workflow%3Atest+branch%3Amaster
+Eventlet was created almost 18 years ago, at a time where async features
```

Comment:
This is important enough that I would either move it to the top, or add a sentence to the top saying "New usages of eventlet are now heavily discouraged, see below for details."

Reply:
I agree, I'll rework that part.